### PR TITLE
[RELAY/OP] Gradient of relay level1 ops

### PR DESCRIPTION
**Pull request by ZihengJiang**

### CODE REVIEW

**Comment by ZihengJiang** [comment]
@MarisaKirisame @reminisce @tqchen @jroesch @yzhliu Please Review

File: python/tvm/relay/op/_tensor_grad.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    return [collapse_sum_like(grad * y, x),
+            collapse_sum_like(grad * x, y)]
+
+
+@register_gradient("divide")
+def divide_grad(orig, grad):
+    """Returns [grad / y,  - grad * (x / y) / y]"""
+    x, y = orig.args
+    return [collapse_sum_like(grad / y, x),
+            collapse_sum_like(- (grad * orig / y), y)]
```
Comment (MarisaKirisame) [inline python/tvm/relay/op/_tensor_grad.py]: I think you have to divide y again here

**Comment by tqchen** [comment]
@kazum @nishi-t please also help to take a look

File: python/tvm/relay/op/_tensor_grad.py (hunk 1 above)
Comment (ZihengJiang) [reply python/tvm/relay/op/_tensor_grad.py]: @MarisaKirisame `orig` is `x / y` here

Comment (MarisaKirisame) [reply python/tvm/relay/op/_tensor_grad.py]: i see

**Comment by MarisaKirisame** [comment]
can we merge this ASAP? i am writing test for ad and i can only test for + and - right now...

**Comment by tqchen** [comment]
@MarisaKirisame, please https://docs.tvm.ai/contribute/code_review.html#approve-and-request-changes-explicitly

**Comment by tqchen** [comment]
Thanks, @ZihengJiang @MarisaKirisame , this is now merged

**Pull request merged**